java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1484,6 +1483,7 @@ public T zrankWithScore(@NonNull String key, @NonNull String member) {
      */
     public T pttl(@NonNull String key) {
         ArgsArray commandArgs = buildArgs(key);
+
```

Comment:
nit: remove newline to be consistent with other methods here

Reply:
I moved this to another PR https://github.com/Bit-Quill/glide-for-redis/pull/201 and reverted here